uploadNeuroDB/tarchiveLoader.pl

```diff
@@ -411,26 +411,10 @@ =head2 Methods
 ################################################################
 ###### Construct the $subjectIDsref array ######################
 ################################################################
-my $subjectIDsref = $utility->determineSubjectID(
-        $scannerID, \%tarchiveInfo, 0, $upload_id
+my ($subjectIDsref) = $utility->determineSubjectID(
+    $scannerID, \%tarchiveInfo, 0, $upload_id, $User, $centerID
 );
 
-################################################################
-###### Get the SessionID #######################################
-################################################################
-if (!defined($subjectIDsref->{'visitLabel'})) { 
-    $subjectIDsref->{'visitLabel'} = 
-    $utility->lookupNextVisitLabel(
-        $subjectIDsref->{'CandID'}, 
-        \$dbh
-    ); 
-}
-my ($sessionID) =
```

Comment:
moved the `getSessionID` function in line 585 where it is needed. Otherwise, queries in that function would fail in case of invalid `CandID`. Since the `SessionID` in this script is only used to updated the `tarchive` and `mri_upload` tables, the call to that function was moved in the same section.